fendermint/vm/interpreter/src/fvm/genesis.rs

```diff
@@ -228,6 +230,26 @@ where
             )
             .context("failed to create reward actor")?;
 
+        // STAGE 1b: Then we initialize the in-repo actors.
+        //
+
+        // Initialize the chain metadata actor which handles saving metadata about the chain
+        // (e.g. block hashes) which we can query.
+        let empty_blockhashes_cid =
+            Array::<(), _>::new_with_bit_width(state.store(), BLOCKHASHES_AMT_BITWIDTH).flush()?;
+        state
+            .create_actor(
+                fendermint_actors::CHAINMETADATA_ACTOR_CODE_ID,
```

Comment:
Thinking of how users could expand the system with their own actors, I really think a unique name would be a better UX than some auto-assigned ID.